### DISCUSSION TITLE
Change weather from celsius to fahrenheit

### DISCUSSION THREAD

**Discussion by gnussbaum67** [end-4/dots-hyprland#602] [General]
Hi,
Great dots by the way!  I was wondering how to change my current weather to Fahrenheit?  I don't have a battery, so I'm getting weather.

**Comment by end-4** [comment]
implemented in 059c893
set `weather.preferredUnit` to `"F"`

**Reply by zeddy303** [reply]
Hey, does this still work for quickshell version? I added this but still shows in the bar and tooltip. Is there a global location for quickshell somewhere? There are a TON of config files have been sifting through trying to find it.

**Reply by end-4** [reply]
set `bar.weather.useUSCS` to `true` in ~/.config/illogical-impulse/config.json

**Reply by zeddy303** [reply]
I did that and now it doesn't update (stays at 0).
UPDATE: It now works. Thank you!
